rdmo/projects/views/project.py

```diff
@@ -109,7 +109,7 @@ def get(self, request, token):
                 error = _('Sorry, your invitation has been expired.')
                 invite.delete()
             elif invite.user and invite.user != request.user:
-                error = _('Sorry, but this invitation is for the user "%s".' % invite.user)
+                error = _('Sorry, but this invitation is for the user "{}".'.format(invite.user))
```

Comment:
This is fine without f-string? Not sure what `_(string)` is actually doing.

Reply:
Think so, the `_` is for the translations, it comes from https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#lazy-translations

Reply:
On a 2nd look, this was actually a sort-of typo in the original code.
Throughout rdmo the lazily translated strings are defined as:
`_('Message from %s') % name`
In this line it was different(typo) and picked up by ruff
`_('Message from %s' % name)`

The recommended `str.format()` style for would be, but is currently not picked up by ruff.
`_('Message from {}').format(name)`

Currently the `.po` files require a `#, python-format` above the `msgid` because of the use of `%s` in the strings.

As long as ruff does not detect the `str.format()` style in lazy strings, I would fix this line by keeping the `%s` .

What do you think about this @jochenklar?

Reply:
added https://docs.astral.sh/ruff/rules/#flake8-gettext-int to our ruff and pushed the fix

Reply:
ok, so in the `gettext` stuff we keep `%s`?

Reply:
yes, exactly, for the `gettext` stuff. The ruff rule should detect this.
And, according to https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#standard-translation, when there are multiple variables it is better to use the named string interpolation instead of positional, like:
` _("Today is %(month)s %(day)s.") % {"month": m, "day": d}`